vsts.yml

```diff
@@ -46,18 +46,49 @@ phases:
      set -e
      echo "===Bootstrapping===" > >(tee -a buildlog.txt) 2> >(tee -a buildlog.txt >&2)
      script/bootstrap > >(tee -a buildlog.txt) 2> >(tee -a buildlog.txt >&2)
+     mkdir s3files
     name: Bootstrap
 
   - bash: |
      set -e
      echo "===Updating for $TARGET_ARCH===" > >(tee -a buildlog.txt) 2> >(tee -a buildlog.txt >&2)
+     export CHROMIUM_BUILDTOOLS_PATH="`pwd`/src/buildtools"
+     echo "Using $CHROMIUM_BUILDTOOLS_PATH as CHROMIUM_BUILDTOOLS_PATH"
      if [ "${COMPONENT}" == "shared_library" ]; then
-       script/update --clean -t $TARGET_ARCH --use-bundled-sccache > >(tee -a buildlog.txt) 2> >(tee -a buildlog.txt >&2)
+       if [ "${TARGET_TYPE}" == "osx" ]; then
```

Comment:
Is this platform check to avoid uploading redundant tars? Perhaps add a comment?